### PR TITLE
Fixes #16819: Add zip support for reports compression

### PR DESCRIPTION
**Pull request by amousset**
https://issues.rudder.io/issues/16819

### CODE REVIEW

**Comment by amousset** [comment]
Commit modified

File: relay/sources/relayd/src/input.rs [hunk 1]
```diff
@@ -54,9 +60,22 @@ pub fn read_compressed_file<P: AsRef<Path>>(path: P) -> Result<Vec<u8>, Error> {
             gz.read_to_end(&mut uncompressed_data)?;
             uncompressed_data
         }
+        Some(Some("zip")) => {
```
Comment (peckpeck) [inline relay/sources/relayd/src/input.rs]: some some :)

Comment (amousset) [reply relay/sources/relayd/src/input.rs]: Fixed!

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
OK, squash merging this PR

**Pull request merged**